src/sceneGraph/index.js

```diff
@@ -386,8 +389,11 @@ createNameSpace("realityEditor.sceneGraph");
         }
     }
 
-    function getWorldPosition(activeKey) {
-        let sceneNode = getSceneNodeById(activeKey);
+    function getWorldPosition(idOrNode) {
```

Comment:
Taking either a string or a scenenode has the potential to cause this function to not be optimized as much as if it were split into getWorldPositionById(id) which calls getWorldPosition(sceneNode) or equivalent

Reply:
Good point. I'll revert to previous implementation for now and defer that refactor for later.